homeassistant/components/ps4/__init__.py

```diff
@@ -141,12 +146,22 @@ def load_games(hass: HomeAssistantType) -> dict:
     """Load games for sources."""
     g_file = hass.config.path(GAMES_FILE)
     try:
-        games = load_json(g_file)
+        games = load_json(g_file, dict)
+    except HomeAssistantError as error:
+        games = {}
+        _LOGGER.error("Failed to load games file: %s", error)
+
+    if not isinstance(games, dict):
+        _LOGGER.error("Games file was not parsed correctly")
+        return {}
```

Comment:
We should just set `games` to an empty dict, right? Not return here already.

Reply:
Changes made. Everything look good?